services/buildbot/master/master.cfg

```diff
@@ -25,15 +25,9 @@ import sys
 sys.path.insert(0, dirname(__file__))
 del expanduser, dirname, sys
 
-from buildbot.changes import filter
 from buildbot.process.properties import Interpolate
 from buildbot.schedulers.timed import Nightly
 from buildbot.schedulers import forcesched
-from buildbot.schedulers.basic import (
-    AnyBranchScheduler,
-    SingleBranchScheduler,
-    )
-from buildbot.status import words, mail
 from buildbot.status.github import GitHubStatus
```

Comment:
Can this be deleted too?  I don't think we've ever had benchmarks report build statuses to GitHub.

Reply:
Glyph would like to get to the point where we can have it tell us about performance regressions. Whether or not that ends up being buildbot or not...